gbbs/benchmarks/BatchDynamicConnectivity/EulerTourTree/ETTree.h

```diff
@@ -591,6 +591,38 @@ struct ETTree {
         return edge_sum;
         //return edge_sum.first ^ edge_sum.second;
     }
+    
+    template <class KY, class VL, class HH>
+    sequence<sequence<sequence<std::pair<uintE, uintE>>>> batch_sub_tree_sum(sequence<std::pair<uintE,uintE>> v_p,
+            pbbslib::sparse_table<KY, VL, HH> edge_index_table)
+    {
+
+
+        size_t num_queries = v_p.size();
```

Comment:
```suggestion
        std::size_t const num_queries = v_p.size();
```
Use namespace descriptions when possible and `const` as much as possible.

Reply:
GOt it !